### ISSUE TITLE
xenial vagrant VM broken

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#3707]
```
    xenial64: You are using pip version 7.1.2, however version 9.0.2 is available.
    xenial64: You should consider upgrading via the 'pip install --upgrade pip' command.
==> xenial64: Running provisioner: build env (shell)...
    xenial64: Running: inline script

    xenial64: /tmp/vagrant-shell: line 1: /home/vagrant/.bash_profile: No such file or directory

    xenial64: Already using interpreter /usr/bin/python3
    xenial64: Using base prefix '/usr'
    xenial64: Traceback (most recent call last):
    xenial64:   File "/usr/local/bin/virtualenv", line 11, in <module>
    xenial64:     sys.exit(main())
    xenial64:   File "/usr/local/lib/python3.5/dist-packages/virtualenv.py", line 832, in main
    xenial64:     symlink=options.symlink)
    xenial64:   File "/usr/local/lib/python3.5/dist-packages/virtualenv.py", line 994, in create_environment
    xenial64:     site_packages=site_packages, clear=clear, symlink=symlink))
    xenial64:   File "/usr/local/lib/python3.5/dist-packages/virtualenv.py", line 1170, in install_python
    xenial64:     mkdir(lib_dir)
    xenial64:   File "/usr/local/lib/python3.5/dist-packages/virtualenv.py", line 448, in mkdir
    xenial64:     os.makedirs(path)
    xenial64:   File "/usr/lib/python3.5/os.py", line 231, in makedirs
    xenial64:     makedirs(head, mode, exist_ok)
    xenial64:   File "/usr/lib/python3.5/os.py", line 231, in makedirs
    xenial64:     makedirs(head, mode, exist_ok)
    xenial64:   File "/usr/lib/python3.5/os.py", line 241, in makedirs
    xenial64:     mkdir(name, mode)

    xenial64: PermissionError: [Errno 13] Permission denied: 'borg-env'
```

**Comment by ThomasWaldmann** [comment]
looks like ubuntu finally switched to using the `vagrant` username, like everybody else (and not `ubuntu`).

**Comment by ThomasWaldmann** [comment]
fixed by #3713
